docs/neo3_guides/compiler_devpack/compilation.md

```diff
@@ -41,12 +13,12 @@ as a Gradle task. Add the following few lines to your `build.gradle`.
 // Add the neow3j gradle plugin to the plugins section.
 plugins {
     id 'java'
-    id 'io.neow3j.gradle-plugin' version "3.3.0"
+    id 'io.neow3j.gradle-plugin' version "3.4.0"
```

Comment:
Can't we use "3.+" here in the `plugins` section? This would avoid updates every single time.

Reply:
ok, i'll change it to 3.+

Reply:
Pay attention because I don't know if this is possible in the `plugins{ }` section. Check it out before changing.

Reply:
It doesn't work for the plugin. But for the other cases i can still change it.